omero/index.rst

```diff
@@ -26,15 +26,15 @@ Additional online resources can be found at:
 
 - :downloads:`Downloads <>`
 - :secvuln:`Security Advisories <>`
-- OMERO API documentation - :javadoc:`OmeroJava API <>`,
-  :pythondoc:`OmeroPy API <>`,
-  :javadoc:`OmeroBlitz / Slice API <slice2html/>`
 - :help:`User help website <>`
 - `OME YouTube channel <https://www.youtube.com/channel/UCyySB9ZzNi8aBGYqcxSrauQ>`_
   for tutorials and presentations
 - `Demo server <http://qa.openmicroscopy.org.uk/registry/demo_account/>`_ -
-  now managed by the main OME team, providing the latest released version of
+  managed by the main OME team, providing the latest released version of
```

Comment:
maybe also mention latest web apps?

Reply:
Pushed. Thoughts? Also, would be quite nice to see _some_ public data on demo. cc: @pwalczysko